store/config.go

```diff
@@ -168,6 +122,19 @@ type Duration struct {
 	time.Duration
 }
 
+// MarshalText returns the byte slice of the formatted duration e.g. "500ms"
+func (d Duration) MarshalText() ([]byte, error) {
```

Comment:
Aren't `MarshalText()` and `UnmarshalText()` for free since we embed `time.Duration`?

Reply:
It results in `"databasePollInterval":{"Duration":500000000}` vs the current implementation of `"databasePollInterval":"500ms"`